.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,35 @@
+name: PyPI release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - run: pip install setuptools wheel
+
+      - run: make install
+
+      - name: Build a binary wheel
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish distribution 📦 to Test PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.test_pypi_password }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
Assuming the test push to Test PyPi was a success, is there any need for this section in the YML now? Or can we condition it on certain pre-release tags or something fancy? 😃

Reply:
Yes, I thought the same, but we haven't done this before. So I thought to deploy to test pypi the same way to normal pypi till we agree how to use that environment.

Reply:
Fair enough! Approving ..